### PR TITLE
International Spanish version

### PR DESCRIPTION
**Pull request by irudoramas**
I took Ealmansi's Spanish translation and made some modifications to convert it from a Latin American Spanish to an international Spanish (I am a professional translator from Canary Islands [Spain]; I am used to producing international Spanish).

I emailed Ealmansi so that he goes over it for the last polishing.

### CODE REVIEW

**Comment by ealmansi** [comment]
Great!

I'll be adding some comments directly on the file you added. You should be able to see them here in a few minutes: https://github.com/bitcoincashorg/website/pull/78/files

I think that for technical reasons and better usability a single international Spanish version would be better. What do you think @zquestz?

File: translations/ES_eu (international Spanish really) [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        - |
+          <strong>Desarrollo descentralizado</strong> - El futuro está a salvo gracias a que son múltiples equipos independientes de desarrolladores los que ofrecen las implementaciones del software. Bitcoin Cash es resistente a ataques políticos y sociales al desarrollo del software. Ningún único grupo o proyecto puede tomar el control. La <a href="https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-ml" target="_blank">lista de correo bitcoin-ml</a> es un buen medio para realizar propuestas de modificaciones al protocolo que requieran coordinación entre los distintos equipos de desarrollo.
+      faq:
+        - question: "¿Qué es Bitcoin Cash?"
+          answer:
+            - |
+              Bitcoin Cash is dinero electrónico de usuario a usuario a través de Internet. Es completamente descentralizado; no existe ningún banco central y funciona sin requerir confianza en ningún tercero.
```
Comment (ealmansi) [inline translations/ES_eu (international Spanish really)]: `is` -> `es`

File: translations/ES_eu (international Spanish really) [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            - |
+              Todas las transacciones de Bitcoin y Bitcoin Cash posteriores a la bifurcación del 1 de agosto son totalmente independientes las unas de las otras. Por este motivo, toda cantidad de Bitcoin adquirida tras la bifurcación no incluye nada de Bitcoin Cash, y viceversa.
+        - question: "¿Cómo se trata el asunto de la repetición de las transacciones entre la cadena antigua y la nueva?"
+          answer:
+            - |
+              Las transacciones de Bitcoin Cash transactions usan un algoritmo de hashing de firmas y vienen indicadas por la etiqueta SIGHASH_FORKID. Estas firmas no son válidas en la red de Bitcoin antigua. Esto evita que las transacciones de Bitcoin Cash se reproduzcan en la cadena de Bitcoin y viceversa.
```
Comment (ealmansi) [inline translations/ES_eu (international Spanish really)]: `transactions` -> ~transactions~

File: translations/ES_eu (international Spanish really) [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        - question: "¿Por qué fue necesaria una bifurcación (<i>fork</i>) para crear Bitcoin Cash?"
+          answer:
+            - |
+              El código de Bitcoin establecía un límite máximo de datos de 1MB por bloque, el equivalente a 3 transacciones por segundo. Aunque aumentar este límite era algo sencillo, la comunidad no pudo alcanzar un consenso a pesar de debatirlo durante años.
+        - question: "¿El límite de 1MB por bloque causaba problemas a Bitcoin?"
+          answer:
+            - |
+              Sí. En 2017 la capacidad alcanzó el "umbral invisible". Las comisiones aumentaron drásticamente y las transacciones de Bitcoin se volvieron poco fiables, dándose el caso de que algunos usuarios no conseguián que sus transacciones se confirmasen incluso tras días de espera.
```
Comment (ealmansi) [inline translations/ES_eu (international Spanish really)]: "Invisible wall" felt OK here to me. Why the change to "invisible threshold"? 

`conseguián` -> `conseguían`

File: translations/ES_eu (international Spanish really) [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            - |
+              El precio de mercado de Bitcoin ha aumentado, pero su crecimiento y utilidad como moneda se han estancado. Muchos usuarios, comerciantes, empresas e incluso inversores abandonaron Bitcoin en busca de alternativas, por lo que el dominio de Bitcoin descendío del 95% hasta alrededor del 40%.
```
Comment (ealmansi) [inline translations/ES_eu (international Spanish really)]: `descendío` -> `descendió`

"a números tan bajos como el 40%" feels more accurate to me than "hasta alrededor del 40%" since that number has increased now.

File: translations/ES_eu (international Spanish really) [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+            - |
+              Estas y otras razones fueron la causa de que fuese necesario implementar Bitcoin Cash como bifurcación previa a la implementación de SegWit. Bitcoin Cash no implementará SegWit.
+        - question: "¿Cuál es el equipo a cargo del desarrollo de Bitcoin Cash?"
+          answer:
+            - |
+              A diferencia de la situación de Bitcoin descrita arriba, en Bitcoin Cash no existe un único equipo de desarrollo. Existen múltiples equipos independientes.
+            - |
+              Esta descentralización del desarrollo (y de las implementaciones de software) constituye un paso adelante muy importante y necesario.
+        - question: "¿Cuál es el símbolo o etiqueta de Bitcoin Cash?"
```
Comment (ealmansi) [inline translations/ES_eu (international Spanish really)]: I think "símbolo bursátil" is in order here, since it's more precise and the direct translation for "ticker".

File: translations/ES_eu (international Spanish really) [hunk 6]
```diff
@@ -0,0 +1,128 @@
+---
+en:
+  title: "Bitcoin Cash - Dinero electrónico de usuario a usuario"
+  meta:
+    description: |
+      Bitcoin Cash es una moneda sólida para todos. Usuarios y comerciantes pueden ahora disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
```
[the remark this answers is not in the document]
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: You are absolutely right

File: translations/ES_eu (international Spanish really) [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+        Bitcoin Cash es una moneda sólida para todos. Usuarios y comerciantes pueden ahora disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
+    og:
+      title: "Bitcoin Cash - Peer-to-Peer Electronic Cash"
+      description: |
+        Bitcoin Cash es una moneda sólida para todos. Usuarios y comerciantes pueden ahora disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
+    misc:
+      logo: "logo"
+    sections:
+      about: "Acerca de"
+      features: "Características"
```
[the remark this answers is not in the document]
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: According to RAE's definition of funcionalidad, it's a characteristic of something. There can't be a list of funcionalidades, at least to our understanding of that word in Spain

File: translations/ES_eu (international Spanish really) [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+      nodes: "Nodos"
+      wallets: "Monederos"
+      services: "Servicios"
+      projects: "Proyectos"
+      exchanges: "Casas de cambio"
+      faq: "Preguntas frecuentes"
+    content:
+      about:
+        - |
+          Bitcoin Cash ofrece una moneda sólida al mundo, cumpliendo con la promesa inicial de Bitcoin como "Dinero electrónico Peer-to-Peer" (de usuario a usuario). Usuarios y comerciantes pueden disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
```
[the remark this answers is not in the document]
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: That's fair, I agree it is inconsistent but I opted to include the original term here as the sentence refers back to Satoshi Nakamoto's creation (plus peer to peer / P2P is a relatively known term in the internet world since torrent downloads came to life). We can remove it for consistency if you consider it doesn't fit there, np.

File: translations/ES_eu (international Spanish really) [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+        - |
+          Todo aquel que poseía Bitcoin antes del bloque 478558 también es poseedor de Bitcoin Cash. Todo el mundo es bienvenido en la comunidad de Bitcoin Cash a medida que avanzamos en la creación de dinero sólido al alcance de todas las personas del planeta.
+      features:
+        - |
+          <strong>Escalabilidad on-chain o en la cadena</strong> - Bitcoin Cash se ciñe a la hoja de ruta de Nakamoto de conseguir la adopción global al escalar <i>on-chain</i> (en la cadena). Como primer paso se ha implementado un límite del tamaño del bloque ajustable, con una capacidad estándar aumentada a 8MB. <a href="https://github.com/BitcoinUnlimited/BUIP/blob/master/065.mediawiki" target="_blank">Se está estudiando el desarollo que permita aumentos masivos en el futuro</a>.
```
[the remark this answers is not in the document]
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: Thanks!
I agree with your suggestion. Se mantiene fiel sounds better than se ciñe.

File: translations/ES_eu (international Spanish really) [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+        - |
+          <strong>Nuevas firmas digitales</strong> - Un nuevo tipo de SigHash proporciona <i>replay protection</i> (protección frente a transacciones repetidas), mayor seguridad en monederos hardware, y la eliminación del problema de hashing cuadrático.
+        - |
+          <strong>Nuevo algoritmo de ajuste de la dificultad (DAA)</strong> - Un ajuste de la dificultad de <i>Proof-of-Work</i> reactiva permite a los mineros migrar desde la cadena antigua de Bitcoin (BTC) según lo deseen, a la vez que ofrece protección frente a fluctuaciones en la tasa de hash.
```
[the remark this answers is not in the document]
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: I opted to leave DAA because it's a known term that shows up in many forums, so readers can identify it. It would make no sense to "create" the siglas AAD out of the blue, visible for the first time in this page. Kind of like GM (general motors) ---> no "MG" in a spanish translation.

Reactivo, you are right, that was a mistake on my part.

Lastly, what do you think about "fluctuaciones en el _hashrate_ (potencia de minado)."

File: translations/ES_eu (international Spanish really) (hunk 3 above)
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: Pared sounds like a physical object, I don't think it makes much sense in Spanish to do a literal translation here. Umbral refers to an upper limit in the form of a digit, a number.

What do you think about "el tope invisible"? (don't know if tope is used commonly in LATAM Spanish, other option is "máximo invisible")

File: translations/ES_eu (international Spanish really) (hunk 4 above)
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: I agree

File: translations/ES_eu (international Spanish really) (hunk 8 above)
[the remark this answers is not in the document]
Comment (ealmansi) [reply translations/ES_eu (international Spanish really)]: No need to remove, makes sense. Also feel free to change "usuario a usuario" if you feel there's a better translation for peer-to-peer. Maybe "par a par" or similar.

File: translations/ES_eu (international Spanish really) (hunk 5 above)
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: I'm really not happy with either of those two options, to be honest.
How about "la abreviatura"? 

Símbolo bursátil sounds way off to me in this context, remember this is addressed to the mainstream public; símbolo bursátil sounds very specific to stocks and trading, which have nothing to do with bitcoin (a currency, not simply an "investment"). Do I make sense?

File: translations/ES_eu (international Spanish really) (hunk 10 above)
[the remark this answers is not in the document]
Comment (ealmansi) [reply translations/ES_eu (international Spanish really)]: I don't feel strongly about the DAA thing but slightly favor AAD. GM seems kind of like a special case since it's a brand and the logo is precisely GM. An example for the opposite would be IMF spelled as FMI in Spanish (USA -> EUA, and other examples).

File: translations/ES_eu (international Spanish really) (hunk 8 above)
[the remark this answers is not in the document]
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: I think usuario a usuario is perfect! "par a par" o "entre pares" sounds strange to the mainstream public

File: translations/ES_eu (international Spanish really) (hunk 10 above)
[the remark this answers is not in the document]
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: I advise strongly against using AAD. During my career as a professional translator, I am never allowed to create new "siglas". Do you agree about removing the parenthesis and the "siglas" entirely?

File: translations/ES_eu (international Spanish really) (hunk 4 above)
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: Wrote "una cifra tan baja como el 40%", do you like?

File: translations/ES_eu (international Spanish really) (hunk 10 above)
[the remark this answers is not in the document]
Comment (ealmansi) [reply translations/ES_eu (international Spanish really)]: OK, we better trust your judgement then :)

File: translations/ES_eu (international Spanish really) (hunk 3 above)
Comment (ealmansi) [reply translations/ES_eu (international Spanish really)]: Uhm. I think the original expression in English itself is somewhat clunky: `(...) capacity hit the 'invisible wall'`. I mean this in the sense that the literal translation in Spanish doesn't feel either more or less clunky to me. If we're not gonna do a literal translation, perhaps `la capacidad alcanzó este límite artificial` would do?

Either way, I find the word `umbral` a bit odd there, because a threshold has a very specific meaning when talking about a system, which is definitely different from the concept of a limit.

**Comment by ealmansi** [comment]
Hey @irudoramas, please also delete the old `es.yml` file in your pull request to avoid duplication.
Alternatively, you can also change the name `es_int` to `es` which will replace the old one.
I think the second alternative is best, at least a long as we don't have multiple `es` versions, since I've already been distributing the `bitcoincash.org/index.es.html` URL today and it will stop working if we don't have an `es.yml` file I believe.

File: translations/ES_eu (international Spanish really) (hunk 3 above)
Comment (irudoramas) [reply translations/ES_eu (international Spanish really)]: límite artificial is perfect. I'm not sure how to edit the file without creating a 4th commit, lol. Replied to you via email

**Comment by zquestz** [comment]
Thank you both!
